pkgs/by-name/iw/iwd/package.nix

```
@@ -13,6 +14,23 @@
   gitUpdater,
 }:
 
+let
+  readline-patch = fetchpatch {
+    url = "https://lists.gnu.org/archive/html/bug-readline/2025-07/txtmA7rksnmmi.txt";
+    hash = "sha256-GIuZWC+dMenBjdPkqQOsX7TI4mUgfPwh/CfzLtTuang=";
+  };
+
+  myreadline = (
+    readline.overrideAttrs (
+      _final: prev: {
+        postPatch = (prev.postPatch or "") + ''
+          patch -p2 < ${readline-patch}
+        '';
```

With `stripLen` in the patch, this can now become:

```suggestion
        patches = (prev.patches or []) ++ [ readline-patch ];
```